paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -179,9 +179,11 @@ class PaparazziPlugin : Plugin<Project> {
         test.inputs.files(nativePlatformFileCollection)
           .withPropertyName("paparazzi.nativePlatform")
           .withPathSensitivity(PathSensitivity.NONE)
+        test.inputs.files(snapshotOutputDir)
+          .withPropertyName("paparazzi.snapshot.output.dir")
+          .withPathSensitivity(PathSensitivity.RELATIVE)
 
         test.outputs.dir(reportOutputDir)
-        test.outputs.dir(snapshotOutputDir)
```

Comment:
I have a feeling this ambiguity comes from the fact that the same task can run in record and verify modes. In record mode the output **is** snapshots, and in verify mode snapshots are input and the output is reports. Does this mean that adding `snapshotOutputDir` as input/output should be based on the `isRecordRun`/`isVerifyRun` properties? (e.g. something like `isRecordRun.map { if (it) snapshotOutputDir else "dummyNonExistentFolder" }`)?